pkg/loadbalancer/redirectpolicy/testdata/address.txtar

```diff
@@ -129,7 +129,7 @@ BE: ID=1 ADDR=10.244.2.1:80/TCP STATE=active
 BE: ID=2 ADDR=[2002::2]:80/TCP STATE=active
 REV: ID=1 ADDR=169.254.169.254:8080
 REV: ID=2 ADDR=[2001::1]:8080
-SVC: ID=1 ADDR=169.254.169.254:8080/TCP SLOT=0 LBALG=undef AFFTimeout=0 COUNT=1 QCOUNT=0 FLAGS=LocalRedirect
+SVC: ID=1 ADDR=169.254.169.254:8080/TCP SLOT=0 LBALG=random AFFTimeout=0 COUNT=1 QCOUNT=0 FLAGS=LocalRedirect
```

Comment:
FIXME: This is not right. We should not annotate unless the annotation is actually set for this service and that the algorithm can be used for this service (e.g. maglev usually not used for ClusterIP)